feat(contributing): add contributing.md, update operating_model (#24)
agntcy/governance: OPERATING_MODEL.md

- **Path to Contribution**:
    1. Submit pull requests or patches.
    2. Engage in discussions, design proposals, and issue triage.
    3. Agree to abide by the [Code of Conduct](CODE_OF_CONDUCT.md) and licensing guidelines.
    4. Be nominated by two (2) or more [committers](#committers) to achieve contributor permissions.

### 3.4 Committers

- Individuals who review and/or approve contributed code, documentation, designs, bug fixes, or moderate community support for The AGNTCY affiliated projects.
- **Path to Committer**:
    1. Frequently review pull requests or patches.
    2. Lead discussions, design proposals, and issue triage.
    3. Be nominated by the AGNTCY Technical Steering Committee to achieve committer permissions.

### 3.5 Technical Steering Committee

The AGNTCY Technical Steering Committee (TSC) is responsible for the governance of projects jointly owned under AGNTCY and its sub-projects. The Technical Steering Committee will set the technical direction of the project and sub-projects.

> - **Composition**: The TSC membership is defined in [CONTRIBUTING.md](CONTRIBUTING.md#technical-steering-committee-members).  It is the intent to establish a voting mechanism and process to expand the Technical Steering Committee, to be determined by contributors. The vision is for the AGNTCY to be a community-governed project and include long-standing contributors and maintainers from affiliated projects.

Reviewer: ```suggestion
- **Composition**: The TSC membership is defined in [CONTRIBUTING.md](CONTRIBUTING.md#technical-steering-committee-members). It is the intent to establish a voting mechanism and process to expand the Technical Steering Committee, to be determined by the Committee itself. The vision is for AGNTCY to be a community-governed project and include long-standing contributors and committers from affiliated projects.
```

### 7.1 Code of Conduct

> - AGNTCY Adopts the [**Contributor Covenant**](https://www.contributor-covenant.org/) (version 2.1 or newer), ensuring a welcoming and respectful community.

Reviewer: ```suggestion
- AGNTCY adopts the [**Contributor Covenant**](https://www.contributor-covenant.org/) (version 2.1 or newer), ensuring a welcoming and respectful community.
```

### Community Growth

> - Number of active sub-projects and affiliate projects that have active contributors, diversity of participation, and new maintainers added.

Reviewer: ```suggestion
- Number of active sub-projects and affiliate projects that have active contributors, diversity of participation, and new committers added.
```

## Conclusion

> This charter and operating model outlines how **The AGNTCY** will operate as a transparent, inclusive, and merit-based open source project. By combining clear governance structures, open contribution pathways, a strong code of conduct, and robust technical processes, The AGNTCY aims to foster a thriving ecosystem and deliver high-quality software solutions through its members.

Reviewer: ```suggestion
This charter and operating model outlines how **AGNTCY** will operate as a transparent, inclusive, and merit-based open source project. By combining clear governance structures, open contribution pathways, a strong code of conduct, and robust technical processes, AGNTCY aims to foster a thriving ecosystem and deliver high-quality software solutions through its members.
```
